app/api/submission/TransitOperation.scala

```diff
@@ -27,6 +27,48 @@ object TransitOperation {
 
   def transform(uA: UserAnswers): TransitOperationType06 =
     uA.metadata.data.as[TransitOperationType06](transitOperationType06.reads(uA.lrn))
+
+  def transformIE013(uA: UserAnswers, mrn: Option[MovementReferenceNumber], flag: Boolean): TransitOperationType04 =
+    uA.metadata.data.as[TransitOperationType04](transitOperationType04.reads(uA.lrn, mrn, flag))
+}
+
+object transitOperationType04 {
+
+  def reads(lrn: String, mrn: Option[MovementReferenceNumber], flag: Boolean): Reads[TransitOperationType04] = (
+    (preTaskListPath \ "declarationType" \ "code").read[String] and
+      (preTaskListPath \ "additionalDeclarationType" \ "code").read[String] and
+      (preTaskListPath \ "tirCarnetReference").readNullable[String] and
+      (preTaskListPath \ "securityDetailsType" \ "code").read[String] and
+      reducedDatasetIndicatorReads and
+      (routeDetailsPath \ "specificCircumstanceIndicator" \ "code").readNullable[String] and
+      (routeDetailsPath \ "routing" \ "bindingItinerary").readWithDefault[Boolean](false) and
+      (transportDetailsPath \ "authorisationsAndLimit" \ "limit" \ "limitDate").readNullable[LocalDate]
+  ).apply {
+    (declarationType,
+     additionalDeclarationType,
+     TIRCarnetNumber,
+     security,
+     reducedDatasetIndicator,
+     specificCircumstanceIndicator,
+     bindingItinerary,
+     limitDate
+    ) =>
+      TransitOperationType04(
+        LRN = if (mrn.isDefined) None else Some(lrn),
+        MRN = mrn.map(_.value),
+        declarationType = declarationType,
+        additionalDeclarationType = additionalDeclarationType,
+        TIRCarnetNumber = TIRCarnetNumber,
+        presentationOfTheGoodsDateAndTime = None, // TODO - do we collect this?
```

Comment:
@Reece-Carruthers Hi Reece, did we need to collect these?

Reply:
@joecefnhedog Hi, that got copied over from the other item in TransitOperation (TransitOperationType06). I just left it on encase it was still relevant - you can just ignore it